src/components/drawing/index.js

```diff
@@ -27,13 +27,67 @@ var DESELECTDIM = require('../../constants/interactions').DESELECTDIM;
 var subTypes = require('../../traces/scatter/subtypes');
 var makeBubbleSizeFn = require('../../traces/scatter/make_bubble_size_func');
 
-var drawing = module.exports = {};
+module.exports = {
+    // data:
+    tester: this.tester,
+    testref: this.testref,
```

Comment:
What's `this` in the case? The module scope function? If so, that's a bad pattern, because it would probably brake things when bundling the code using bundler that does not scope every module into its own function.

Reply:
Good catch.
Fixed in https://github.com/plotly/plotly.js/pull/4104/commits/e9fe58ccb74f4cfce33d9973c7f4925dd1f77127.